modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,420 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-ocm-role.adoc
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-sts-aws-requirements-attaching-boundary-policy_{context}"]
+= Attaching a boundary policy to an IAM role
+
+You can apply a policy as a _permissions boundary_ on a role.
+You can use an AWS-managed policy or a customer-managed policy to set the boundary for an IAM entity (user or role). That policy limits the maximum permissions for the user or role.
+
+When using the boundary policy files:
+
+* *core + VPC* = create VPC permissions are enabled
+** Installer cannot create or manage the Privatelink (PL).
+** Assumes you are not using BYO-VPC.
+** Assumes the installer is using create or manage the VPC.
+
+* *core + PrivateLink (PL) + VPC* = Privatelink (PL) permissions are enabled
+** Assumes you want to use PL for a private cluster and also requires BYO-VPC.
```

Comment:
I believe this is incorrect - if they're providing VPC permissions, then we would _not_ expect a BYOVPC cluster.